openfisca_aotearoa/entities.py

```diff
@@ -3,35 +3,17 @@
 # This file defines the entities needed by our legislation.
 from openfisca_core.entities import build_entity
 
+# General note about how to consider roles as they are utilised in OpenFisca Aotearoa
+#
+# Best practice would be to not utilise roles as legal definitions, instead as a framework for structuring scenarios.
+#
+# For instance, a person in the role of child (family entity) should not be assumed to be a social_security__dependent_child
+# or even a social_security__child. Those legal concepts should be required of the persons put in the child role.
+#
+# In some more generic situations the role can be used to imply a more general concept. For an example of this see: person_has_partner which will return true if the person occupys the role of partner in a scenario.
+# The decision to lean on this variable should be carefully considered in light of the definitions within the instrument of law being interpreted.
+# If the instrument has it's own definition for partner - it's probably better to fashion a variable that includes the appropriate tests to that use case.
 
-Titled_Property = build_entity(
```

Comment:
Camel Case?

Reply:
This was just moved rather than created.